Documentation/ApiOverview/ContentSecurityPolicy/_csp_example_com.yaml

```diff
@@ -0,0 +1,23 @@
+# Inherits default frontend policy mutations provided by Core and 3rd-party extensions (enabled per default)
+inheritDefault: true
+mutations:
+  # Allow frames/iframes to anywhere (HTTPS only, not HTTP)
+  # (Note: it's "frame-src" not "iframe-src")
+  - mode: "extend"
+    directive: "frame-src"
+    sources:
+      - "https:"
```

Comment:
At least for `<frame>` I'd be in favor of listing only the allowed (trusted) domains (e.g. `https://*.example.org` and `https://*.example.com`).

Reply:
Yes. Good catch and thanks for bringing this up. Changed.